### PR TITLE
Meetings 2024 Feb/March

### PR DESCRIPTION
**Pull request by KathleenDollard**
We are making a procedural change to better manage the backlog.
 
I am scheduling quite a ways out and need input from everyone on the schedule: 
 
* Can your item be delayed past its current scheduled date? We have tough scheduling choices, so lets make sure we are prioritizing the right things.
* Do I have the correct time allotted. One item means two hours, two items are 1 hour each, and comment if you think your item is less than 1 hour?
* Will the right people be available on the scheduled date?
* If work will be blocked if we can't scheduled something earlier, comment, but with the missed meetings, there are no great answers.
* Please get links in as early as possible, definitely at least a week ahead of time.

### CODE REVIEW

File: meetings/2024/README.md [hunk 1]
```diff
@@ -39,18 +29,35 @@ All schedule items must have a public issue or checked-in proposal that can be l
 
 ### Mon Apr 1, 2024
 
+- [Implementation specific documentation](https://github.com/dotnet/csharplang/issues/7898). (Bill, on a Monday)
```
Comment (BillWagner) [inline meetings/2024/README.md:32]: Can we move this to a different Monday?  Easter Monday is a holiday in the UK, and committee members may not be available.

File: meetings/2024/README.md [hunk 2]
```diff
@@ -39,18 +29,35 @@ All schedule items must have a public issue or checked-in proposal that can be l
...
+- Extensions (Julien and Mads)
+
 ### Mon Feb 26, 2024
 
+- Review [metadata](https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#metadata) section for"Params Collections". (Aleksey)
+- Review: [Allow lambdas with parameter modifiers, but without parameter types](https://github.com/dotnet/csharplang/pull/7369) (Cyrus)
+
 ### Wed Feb 21, 2024
 
+- Dictionary-expressions + Collection-expressions future: WG to go over our plans, and the initial design space (Cyrus)
```
Comment (cston) [inline meetings/2024/README.md:59]: I'd like to attend for this topic if possible, but I cannot make it 2/21.

Comment (cston) [reply meetings/2024/README.md:59]: Could we consider switching the agenda items for 2/21 and 2/26, if that works for @CyrusNajmabadi and @AlekseyTs?

Comment (CyrusNajmabadi) [reply meetings/2024/README.md:59]: Works for me.

File: meetings/2024/README.md (hunk 1 above)
Comment (KathleenDollard) [inline meetings/2024/README.md:32]:
```suggestion

```

File: meetings/2024/README.md [hunk 3]
```diff
@@ -4,18 +4,8 @@ All schedule items must have a public issue or checked-in proposal that can be l
 
 ## Schedule ASAP
 
-- Review [metadata](https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#metadata) section for"Params Collections". (Aleksey)
-- 2 hours please. Dictionary-expressions + Collection-expressions future: WG to go over our plans, and the initial design space (Cyrus)
-- Review: Allow lambdas with parameter modifiers, but without parameter types: https://github.com/dotnet/csharplang/pull/7369 (Cyrus)
-
 ## Schedule when convenient
```
Comment (KathleenDollard) [inline meetings/2024/README.md:8]:
```suggestion

- [Implementation specific documentation](https://github.com/dotnet/csharplang/issues/7898). (Bill, on a Monday)
```

File: meetings/2024/README.md (hunk 2 above)
Comment (KathleenDollard) [reply meetings/2024/README.md:59]: @AlekseyTs will Feb 26 work for you?

**Comment by AlekseyTs** [comment]
>will Feb 26 work for you?

For what topic? I can present my topic(s) on either date.

File: meetings/2024/README.md [hunk 4]
```diff
@@ -37,24 +27,43 @@ All schedule items must have a public issue or checked-in proposal that can be l
...
 ### Wed Feb 28, 2024
 
+- Extensions (Julien and Mads)
+
 ### Mon Feb 26, 2024
 
+- Review [metadata](https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#metadata) section for"Params Collections". (Aleksey)
+- Review: [Allow lambdas with parameter modifiers, but without parameter types](https://github.com/dotnet/csharplang/pull/7369) (Cyrus)
+
 ### Wed Feb 21, 2024
```
Comment (KathleenDollard) [inline meetings/2024/README.md:59]:
```suggestion
### Wed Feb 26, 2024
```

File: meetings/2024/README.md [hunk 5]
```diff
@@ -37,24 +27,43 @@ All schedule items must have a public issue or checked-in proposal that can be l
...
 
+- Dictionary-expressions + Collection-expressions future: WG to go over our plans, and the initial design space (Cyrus)
```
Comment (KathleenDollard) [inline meetings/2024/README.md:61]:
```suggestion
- Dictionary-expressions + Collection-expressions future: WG to go over our plans, and the initial design space (Cyrus)

### Mon Feb 21, 2024

- Review [metadata](https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#metadata) section for"Params Collections". (Aleksey)
- Review: [Allow lambdas with parameter modifiers, but without parameter types](https://github.com/dotnet/csharplang/pull/7369) (Cyrus)

```

File: meetings/2024/README.md [hunk 6]
```diff
@@ -37,24 +27,43 @@ All schedule items must have a public issue or checked-in proposal that can be l
...
+- [Overload resolution priority](https://github.com/dotnet/csharplang/pull/7906) (Fred)
+
 ### Wed Feb 28, 2024
 
+- Extensions (Julien and Mads)
+
 ### Mon Feb 26, 2024
 
+- Review [metadata](https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#metadata) section for"Params Collections". (Aleksey)
+- Review: [Allow lambdas with parameter modifiers, but without parameter types](https://github.com/dotnet/csharplang/pull/7369) (Cyrus)
```
Comment (KathleenDollard) [inline meetings/2024/README.md:57]:
```suggestion
```

File: meetings/2024/README.md [hunk 7]
```diff
@@ -37,24 +27,45 @@ All schedule items must have a public issue or checked-in proposal that can be l
...
 
+- [Non-enumerable collection types](https://github.com/dotnet/csharplang/pull/7895) (Julien)
+- [Overload resolution priority](https://github.com/dotnet/csharplang/pull/7906) (Fred)
+
 ### Wed Feb 28, 2024
 
-### Mon Feb 26, 2024
+- Extensions (Julien and Mads)
+
+
```
Comment (KathleenDollard) [inline meetings/2024/README.md:54]:
```suggestion
```

File: meetings/2024/README.md [hunk 8]
```diff
@@ -41,24 +33,39 @@ All schedule items must have a public issue or checked-in proposal that can be l
...
 
-### Wed Feb 21, 2024
+- Extensions (Julien and Mads)
 
-### Mon Feb 5, 2024 (unvisited topics)
+### Wed Feb 26, 2024
 
 - Dictionary-expressions + Collection-expressions future: WG to go over our plans, and the initial design space (Cyrus)
 
+### Mon Feb 21, 2024
```
Comment (KathleenDollard) [inline meetings/2024/README.md:64]:
```suggestion
### Wed Feb 21, 2024
```

File: meetings/2024/README.md [hunk 9]
```diff
@@ -41,24 +33,39 @@ All schedule items must have a public issue or checked-in proposal that can be l
...
+- [Overload resolution priority](https://github.com/dotnet/csharplang/pull/7906) (Fred)
 
-### Mon Feb 26, 2024
+### Wed Feb 28, 2024
 
-### Wed Feb 21, 2024
+- Extensions (Julien and Mads)
 
-### Mon Feb 5, 2024 (unvisited topics)
+### Wed Feb 26, 2024
```
Comment (KathleenDollard) [inline meetings/2024/README.md:60]:
```suggestion
### Mon Feb 26, 2024
```

File: meetings/2024/README.md [hunk 10]
```diff
@@ -4,20 +4,12 @@ All schedule items must have a public issue or checked-in proposal that can be l
 
 ## Schedule ASAP
 
-- Review [metadata](https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#metadata) section for"Params Collections". (Aleksey)
-- 2 hours please. Dictionary-expressions + Collection-expressions future: WG to go over our plans, and the initial design space (Cyrus)
-- Review: Allow lambdas with parameter modifiers, but without parameter types: https://github.com/dotnet/csharplang/pull/7369 (Cyrus)
-- Open question: [`params` mismatches across inheritance](../../proposals/params-collections.md#consider-enforcing-scoped-or-params-across-overrides) (Fred, Aleksey)
+## Schedule when convenient
```
Comment (cston) [inline meetings/2024/README.md:7]: >Schedule when convenient [](http://example.com/codeflow?start=3&length=24)

This title is duplicated below.

**Review by cston — APPROVED** [review]

File: meetings/2024/README.md [hunk 11]
```diff
@@ -4,20 +4,12 @@ All schedule items must have a public issue or checked-in proposal that can be l
...
+
 - Open question: [Should presence of required members prevent declaration of params parameter?](https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#should-presence-of-required-members-prevent-declaration-of-params-parameter) (Aleksey)
```
Comment (AlekseyTs) [inline meetings/2024/README.md:9]: This was supposed to be a "Schedule ASAP" item.

**Pull request merged**